docker/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.5"
+networks:
+  default:
+    name: supernets2
+services:
+  supernets2-mock-l1-network:
+    container_name: supernets2-mock-l1-network
+    image: snapchain/geth-cdk-validium-contracts
+    healthcheck:
+      test: [ "CMD-SHELL", "geth attach --datadir /geth_data --exec eth.blockNumber" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    ports:
+      - 8545:8545
+      - 8546:8546
+    command:
+      - "--http"
+      - "--http.api"
+      - "admin,eth,debug,miner,net,txpool,personal,web3"
+      - "--http.addr"
+      - "0.0.0.0"
+      - "--http.corsdomain"
+      - "*"
+      - "--http.vhosts"
+      - "*"
+      - "--ws"
+      - "--ws.origins"
+      - "*"
+      - "--ws.addr"
+      - "0.0.0.0"
+      - "--dev"
+      - "--datadir"
+      - "/geth_data"
+      - "--syncmode"
+      - "full"
+      - "--rpc.allow-unprotected-txs"
+
+  dac-setup-committee:
+    container_name: dac-setup-committee
+    image: snapchain/dac-setup-committee
+    depends_on:
+      supernets2-mock-l1-network:
+        condition: service_started
+    volumes:
+      - ./deploymentOutput/deploy_output.json:/cdk-validium-contracts/docker/scripts/deploy_output.json          
+    environment:
+      - JSONRPC_HTTP_URL=http://supernets2-mock-l1-network:8545
+      - DAC_URL=http://supernets2-data-availability:8444
```

Comment:
I don't see a service named `supernets2-data-availability` in this compose file. how does it work?

Reply:
the code could connect the local geth which another container is running and this test doesn't need a DAC service.